File: 0884-uncommon-words-from-two-sentences/0884-uncommon-words-from-two-sentences.js
/**
 * @param {string} s1
 * @param {string} s2
 * @return {string[]}
 */
var uncommonFromSentences = function(s1, s2) {
    let se1=s1.split(' ')
    let se2=s2.split(' ')
    let sentance=[...se1,...se2]
    let obj={}
    let arr=[]
    for(i=0;i<sentance.length;i++){
        value=sentance[i]
        if(obj[value]){
            obj[value]++
        }
        else{
            obj[value]=1
        }
    }
    for (let key in obj ){
        if (obj[key]==1){
            arr.push(key)
        }
    }  
    return arr
};
console.log(uncommonFromSentences("this apple is sweet", "this apple is sour"));

File: 0884-uncommon-words-from-two-sentences/0884-uncommon-words-from-two-sentences.py
class Solution:
    def uncommonFromSentences(self, s1: str, s2: str) -> List[str]:
        se1=s1.split()
        se2=s2.split()
        sentance=se1+se2
        count={}
        res=[]
        for x in range(len(sentance)):
            value=sentance[x]
            if value in count:
                count[value]+=1
            else:
                count[value]=1
        for key in count:
            if count[key]==1:
                res.append(key)
        return res

        